### PR TITLE
EPMDJ-6740: Add buttons classes

### PR DESCRIPTION
**Pull request by IliaSolovev**
Issue: EPMDJ-6740

### CODE REVIEW

File: src/css/button.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    color: map-get($monochrome, white);
+  }
+}
+
+button {
+  @extend .button-primary;
+}
+
+.button-secondary {
+  @extend .button-primary;
```
Comment (RomanDavlyatshin) [inline src/css/button.scss:41]: ❤️ 👍

File: src/pages/styles.module.scss [hunk 2]
```diff
@@ -4,7 +4,6 @@
  * CSS files with the .module.css suffix will be treated as CSS modules
  * and scoped locally.
  */
-
 .heroBanner {
```
Comment (RomanDavlyatshin) [inline src/pages/styles.module.scss]: Why is it here 😮 ❓ @IliaSolovev  (I believe you've committed that by a mistake and that should be placed elsewhere)

**Review by RomanDavlyatshin — CHANGES_REQUESTED** [review]

**Review by RomanAchilov — APPROVED** [review]

File: docs/usage-exapmle.md [hunk 3]
```diff
@@ -36,7 +37,13 @@ fun main(args: Array<String>) {
 ```ts
   const greeting = () => console.log("Hello, world!");
 ```
-<button>Try demo</button>
-<button className="button-secondary">Try demo</button>
-<button className="button-ghost">Try demo</button>
+<button className="button-primary">
+    <img src="../static/img/download-icon.svg" alt=""/>
```
Comment (RomanDavlyatshin) [inline docs/usage-exapmle.md]: Please, do not use `` tag for icons
I suggest implementing `<Icon src="../static/img/download-icon.svg">` component

File: docs/usage-exapmle.md [hunk 4]
```diff
@@ -36,7 +37,13 @@ fun main(args: Array<String>) {
...
+    Try demo
+</button> <br/>
+<button className="button-secondary">
+    <Download/>
```
Comment (RomanDavlyatshin) [inline docs/usage-exapmle.md]: Don't import `svg` icons into the page, as they will be indexed as part of the site content. (Use `<Icon>` component and display `svg` via `background-image` instead).

**Review by RomanDavlyatshin — CHANGES_REQUESTED** [review]

**Pull request merged**